src/components/picker/picker.tsx

```diff
@@ -99,6 +97,10 @@ export class Picker {
 
     /**
      * True if the dropdown list should be displayed without cutting the content
+     *
+     * @deprecated This was used for a workaround, and isn't needed any
+     * longer. Setting it has no effect, and the property will be removed
+     * in the next major version.
```

Comment:
We don't need to set this commit as a breaking change, since the functionality was broken in an earlier version, and all we do in this PR is fixing it the bug, albeit in a way that makes the property unnecessary.

The breaking change comes when we actually remove the property.

I can update the commit message, since I have the branch checked out anyway.

Reply:
ah okey, thanks! 🙂

Reply:
⚡